src/components/Form/DocumentSelection.jsx

```diff
@@ -29,9 +29,9 @@ const DocumentSelection = ({ htmlForAndIdProp, handleDocType, docType }) => (
         onChange={handleDocType}
         name="document"
       >
-        {docTypes.map((doc) => (
-          <MenuItem key={doc.split(' ')[0]} value={doc}>
-            {doc}
+        {Object.entries(DOC_TYPES).map(([key, value]) => (
+          <MenuItem key={key} value={key}>
```

Comment:
A bit confused as to why `key` is used for value instead of `value`. I'll patch this up.

Reply:
Made PR for minor fix (PR #306)